zerver/webhooks/netlify/doc.md

```diff
@@ -0,0 +1,16 @@
+See your Netlify deployment notifications in Zulip!
+
+1. {!create-stream.md!}
+
+2. {!create-bot-construct-url-indented.md!}
+
+3. To add a webhook, go to Settings -> Build & deploy -> Outgoing notifications -> Add Notification
+   Choose to Outgoing webhook and the event you want to listen for. For the URL to
+   notify, copy and paste the URL above.
+
+    For more information regarding Netlify webhooks, see:
+    [Netlify's webhook documentation][1].
+
+[1]: https://www.netlify.com/docs/webhooks/
+
```

Comment:
I would recommend reading [this guide](https://chat.zulip.org/api/integration-docs-guide), and making sure that this file conforms to the guide's standards! :)